src/main/java/org/springframework/social/salesforce/api/Salesforce.java

```diff
@@ -39,6 +39,8 @@
 
     public UserOperations userOperations();
 
+    public UserOperations userOperations(String gatewayUrl);
```

Comment:
It would need to remain the same for the entire time that the connection was instantiated for. Each time you request a new connection a new API Adaptor is instantiated.